tests/rptest/tests/pandaproxy_test.py

```diff
@@ -289,12 +289,14 @@ def _create_named_consumer(self,
         return res
 
 
-class PandaProxyTest(PandaProxyEndpoints):
+class PandaProxyTestMethods(PandaProxyEndpoints):
     """
-    Test pandaproxy against a redpanda cluster.
+    Base class for testing pandaproxy against a redpanda cluster.
+
+    Inherit from this to run the tests.
     """
-    def __init__(self, context):
-        super(PandaProxyTest, self).__init__(context)
+    def __init__(self, context, **kwargs):
+        super(PandaProxyTestMethods, self).__init__(context, **kwargs)
```

Comment:
> dt/pandaproxy: Run all tests with auto-auth

i'm not exactly seeing in this commit what change makes all the tests run with auto-auth. what am i missing?

Reply:
i got that, i just didn't see where the new base class enabled auto auth. but now i realize i think you introduced that back in a previous pr.